src/server/status/metrics.js

```diff
@@ -1,11 +1,14 @@
 import _ from 'lodash';
 import Samples from './samples';
-module.exports = function (kbnServer, server, config) {
-  let lastReport = Date.now();
+import { keysToSnakeCaseShallow } from '../../utils/case_conversion';
 
+export function collectMetrics(kbnServer, server, config) {
+  let lastReport = Date.now();
   kbnServer.metrics = new Samples(12);
 
   server.plugins['even-better'].monitor.on('ops', function (event) {
+    kbnServer.v6Metrics = getV6Metrics({ event, config });
```

Comment:
What are your thoughts on swapping this, creating `kbnServer.legacyMetrics`. Reason being is we are exposing the cgroup information via this interface.